pulse2percept/implants.py

```diff
@@ -253,10 +262,10 @@ def receptive_field(self, xg, yg, rftype='square', size=None):
 class ElectrodeArray(object):
 
     def __init__(self, etype, radii, xs, ys, hs=0, names=None):
-        """Create an ElectrodeArray on the retina
+        """Creates an ElectrodeArray on the retina
 
-        This function creates an electrode array of type `etype` and places it
-        on the retina. Lists should specify, for each electrode, its size
+        This function adds one or more electrodes to an existing ElectrodeArray
```

Comment:
Good catch! I accidentally copied too much of the docs from the `add_electrodes` method...